CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 #### [v5.4.0](https://github.com/opengovsg/FormSG/compare/v5.3.0...v5.4.0)
 
 - refactor: remove unnecessary field validator JS tests in favour of TS tests [`#1475`](https://github.com/opengovsg/FormSG/pull/1475)
-- feat: revert custom code for CorpPass cloud migration [`#1476`](https://github.com/opengovsg/FormSG/pull/1476)
+- feat: revert custom code for Corppass cloud migration [`#1476`](https://github.com/opengovsg/FormSG/pull/1476)
```

Comment:
Might want to revert the casings here, since this is a changelog (and will get automatically updated the next time we release anyways)